core/jreleaser-model-impl/src/main/java/org/jreleaser/model/internal/deploy/maven/MavenDeployer.java

```diff
@@ -107,6 +107,10 @@ public interface MavenDeployer<A extends org.jreleaser.model.api.deploy.maven.Ma
 
     List<String> keysFor(String property);
 
+    boolean getMayDeploySnapshots();
```

Comment:
Remove this method as the getter `isSnapshotSupported` is inherited from `Activatable`.